bin/fetch-configlet.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+import platform
+import re
+import sys
+from fetch import fetch
+
+LATEST = 'https://github.com/exercism/configlet/releases/latest'
+
+try:
+    # Python 3
+    from urllib.request import urlopen
+except ImportError:
+    # Python 2
+    from urllib2 import urlopen
+
+
+def get_os():
+    os = platform.system()
+    if os.startswith('Darwin'):
+        return 'mac'
+    elif os.startswith('Windows'):
+        return 'windows'
+    else:
+        return 'linux'
+
+
+def get_arch():
+    arch = platform.machine()
+    if '64' in arch:
+        return '64bit'
+    elif '686' in arch:
+        return '32bit'
+    elif '386' in arch:
+        return '32bit'
+    else:
+        return '64bit'
```

Comment:
`configlet` also supports ARM. On ARMv7 `platform.machine() == "armv7l"` (at least on my Raspberry Pi 3) - the ARMv6 build seems to work find on this anyway. I'm not sure about ARMv8, since that's 64-bit but can apparently run the 32-bit code from earlier versions in some cases - I don't have a spare SD card at the moment to test it with my Pi, so I would suggest 

Feel free to tidy it up however you want, but this section should work:

```python
...
elif arch.startswith("armv5"):
    return "arm-v5"
elif arch.startswith("armv6") or arch.startswith("armv7"):
    return "arm-v6"
...
```

Reply:
I guess I shouldn't have assumed that the existing `fetch-configlet` script was covering all supported architectures, since it clearly wasn't tested on all of them. I'll add this section.